wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/templates/sample/Robot.java

```diff
@@ -99,7 +99,7 @@ public void autonomous() {
 		switch (autoSelected) {
 			case kCustomAuto:
 				m_robotDrive.setSafetyEnabled(false);
-				m_robotDrive.arcadeDrive(-0.5, 1.0); // spin at half speed
+				m_robotDrive.arcadeDrive(0.0, 0.5); // spin at half speed
 				Timer.delay(2.0); // for 2 seconds
 				m_robotDrive.arcadeDrive(0.0, 0.0); // stop robot
```

Comment:
This doesn't match the docs above. The docs above have driving forwards at half speed, then driving forward at full speed for 3 seconds.